README.md

```diff
@@ -1,2 +1,51 @@
-Hello-World
-In this project, you will set up a "Hello world" repository.
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-world
+
+> This project is my first ever on microverse and its a linter setup as well as a hello world
+
+![screenshot](./app_screenshot.png)
+
+This is a simple hello world project
+
+## Built With
+
+- HTML
+- CSS
+
+<!-- ## Live Demo
+
+[Live Demo Link](https://livedemo.com) -->
+
+
+## Getting Started
+
+
+you can clone and play around 
```

Comment:
- [ ] In the `Instructions setting up your project locally` section. You are supposed to give detailed information to someone interested in your project. For example, how and what to clone instructions

Reply:
- [ ] As requested by one of the previous reviewers, I will encourage you to add more details to the `Gettng started` so that users can be able to run the project locally.  Something similar to the below example:

```
To get a local copy up and running follow these simple example steps.

- clone the repository using `https://github.com/enaburekhan/Review-past-project.git` to your local machine
- Install live server
- Right click on the index.html and open with live server.
```
You can change the content to suit your project. This will make your readme more descriptive.